enterprise/server/workflow/service/service.go

```diff
@@ -501,10 +501,13 @@ func (ws *workflowService) ExecuteWorkflow(ctx context.Context, req *wfpb.Execut
 	}
 
 	if req.GetClean() {
-		err = ws.useCleanWorkflow(ctx, wf)
+		// Set a unique instance name on the workflow so this run is in a clean
+		// container
+		suffix, err := random.RandomString(10)
```

Comment:
Decided to go a different approach: https://github.com/buildbuddy-io/buildbuddy/pull/6303

Discussed with Siggi and we thought it may be confusing to have multiple buttons with nuanced snapshot invalidation behavior. The most intuitive is that the button only invalidates the snapshot for the current invocation